data/etc/plinth/modules-enabled/diaspora

```diff
@@ -1,2 +0,0 @@
-plinth.modules.diaspora
```

Comment:
We have a record of the disabled apps in `DISABLED_APPS_TO_REMOVE` in `setup.py`. So, I think it's okay to delete the file.

I think we should track what packages are disabled in the Plinth source itself, for easier reference/modification for developers. 
And, I actually don't know how to give a pull request in Alioth. 🤐